python2/httplib2test.py

```diff
@@ -105,7 +100,7 @@ def test(self):
         self.assertEqual(233, len(httplib2.safename(uri2)))
         self.assertEqual(233, len(httplib2.safename(uri)))
         # Unicode
-        if sys.version_info >= (2,3):
+        if sys.version_info >= (2, 7):
```

Comment:
For `python2/httplib2test.py` there is no other version except 2.7, so please remove this condition.